BMM.UI.iOS/Application/NewMediaPlayer/IosMediaPlayer.cs

```diff
@@ -121,6 +121,12 @@ public async Task Play(IList<IMediaTrack> mediaTracks, IMediaTrack currentTrack,
             }
         }
 
+        public async Task PrepareToPlay(IList<IMediaTrack> mediaTracks, IMediaTrack currentTrack, string playbackOrigin, long startTimeInMs = 0)
```

Comment:
PrepareToPlay basically sets up the queue so that the user just needs to hit play to continue where he left off, right?
It's hard to name. Is RecoverQueue() a better name?